front/app/containers/ProjectsShowPage/shared/header/ProjectActionButtons.tsx

```diff
@@ -131,55 +132,42 @@ const ProjectActionButtons = memo<Props>(({ projectId, className }) => {
 
   const { process_type, publication_status } = project.data.attributes;
 
-  const isProjectArchived =
-    project.data.attributes.publication_status === 'archived';
+  const isProjectArchived = publication_status === 'archived';
   const isProcessTypeContinuous = process_type === 'continuous';
-  const participation_method = isProcessTypeContinuous
+  const participationMethod = isProcessTypeContinuous
     ? project.data.attributes.participation_method
     : currentPhase?.attributes.participation_method;
   const ideas_count = isProcessTypeContinuous
     ? project.data.attributes.ideas_count
     : currentPhase?.attributes.ideas_count;
-  const hasProjectEnded = currentPhase
+  // For a continuous project, hasCurrentPhaseEnded will always return false.
+  const hasCurrentPhaseEnded = currentPhase
     ? pastPresentOrFuture([
         currentPhase.attributes.start_at,
         currentPhase.attributes.end_at,
       ]) === 'past'
     : false;
-  const inputTerm = getInputTerm(
-    project.data.attributes.process_type,
-    project.data,
-    phases?.data
-  );
-
-  const isParticipationMethodIdeation = participation_method === 'ideation';
-
+  const inputTerm = getInputTerm(process_type, project.data, phases?.data);
   const isParticipationMethodNativeSurvey =
-    participation_method === 'native_survey' ||
-    currentPhase?.attributes.participation_method === 'native_survey';
+    participationMethod === 'native_survey';
+  const isParticipationMethodIdeation = participationMethod === 'ideation';
 
   const showSeeIdeasButton =
-    ((isProcessTypeContinuous && isParticipationMethodIdeation) ||
-      currentPhase?.attributes.participation_method === 'ideation') &&
-    isNumber(ideas_count) &&
-    ideas_count > 0;
-
-  const showIdeasButton =
-    isParticipationMethodIdeation && publication_status !== 'archived';
+    isParticipationMethodIdeation && isNumber(ideas_count) && ideas_count > 0;
 
-  const showNativeSurvey =
-    isParticipationMethodNativeSurvey && publication_status !== 'archived';
+  const showPostIdeaButton =
+    !isProjectArchived && isParticipationMethodIdeation;
 
-  const showSurvey =
-    (!isProjectArchived && phases && participation_method === 'survey') ||
-    (currentPhase?.attributes.participation_method === 'survey' &&
-      !hasProjectEnded);
+  const showTakeNativeSurveyButton =
+    !isProjectArchived && isParticipationMethodNativeSurvey;
 
-  const showPoll =
-    ((isProcessTypeContinuous && participation_method === 'poll') ||
-      currentPhase?.attributes.participation_method === 'poll') &&
-    !hasProjectEnded;
+  const showTakeSurveyButton =
+    !isProjectArchived &&
+    participationMethod === 'survey' &&
+    !hasCurrentPhaseEnded;
 
+  const showTakePollButton =
+    participationMethod === 'poll' && !hasCurrentPhaseEnded;
```

Comment:
This should probably also be hidden if project is archived. The button shows atm.

Reply:
I started to make changes like this, but got into a rabbit hole so I decided to not make functional changes. cc @amanda-anderson